src/bin/cargo/cli.rs

```diff
@@ -406,14 +406,12 @@ impl GlobalArgs {
 pub fn cli() -> Command {
     let is_rustup = std::env::var_os("RUSTUP_HOME").is_some();
     let usage = if is_rustup {
-        "cargo [+toolchain] [OPTIONS] [SUBCOMMAND]"
+        "cargo [+toolchain] [OPTIONS] [COMMAND]"
```

Comment:
This was changed to be consistent with the rest of clap.  We can instead tell clap to use `SUBCOMMAND` if we want.

Personally, I think what we use is context-dependent.  Every subcommand is a command, just in a subordinate position to another command.  Using "command" is appropriate its just some places it can be useful to clarify that it is subordinate.

Reply:
Make sense! I personally agree with your point and Eric's (need to scrub everything only for consistency).